src/screens/home/Home.js

```diff
@@ -25,315 +35,357 @@ import { SvgXml } from "react-native-svg";
-          </Section>
-          <VStack>
-            {vehiclesListArr}
-            {this.state.edit && (
-              <IconButton
-                leftMargin={4}
-                rightMargin={4}
-                text="Add Vehicle"
-                onPress={() => this._addVehicleSection()}
-                icon={<Icon color="white" name="local-taxi" size={50} />}
-              />
-            )}
-          </VStack>
-          {this.state.edit ? (
-            <Section title="Non-motorists">
-              <HStack flexWrap="wrap">{nonmotoristListArr}</HStack>
-              <IconButton
-                topMargin={4}
-                text="Add Non-Motorist"
-                onPress={() => this._addNonmotorist()}
-                icon={<Icon color="white" name="person-add" size={50} />}
-              />
-            </Section>
-          ) : (
-            <>
-              {nonmotoristListArr.length ? (
-                <Section title="Non-motorists">
-                  <HStack flexWrap="wrap">{nonmotoristListArr}</HStack>
-                </Section>
-              ) : null}
-              <Section title="Crash Diagram">
-                {photo.image ? (
+      // render the right view
+      return (
+         <SafeAreaView flex={1}>
+            {rightControls()}
+            <ScrollView marginBottom={24}>
+               <Section title="Crash and Roadway">
+                  {this.state.edit ? (
+                     <HStack alignItems="center" mt={4}>
+                        <Icon name="edit-road" size={50} />
+                        <Text ml={2}>Crash/Road</Text>
+                     </HStack>
+                  ) : (
+                     <HStack>
+                        <IconButton
+                           topMargin={4}
+                           text={`Crash/Road \nForm`}
+                           onPress={() =>
+                              navigateQuestion(
+                                 roadForm,
+                                 road.data[0].id,
+                                 "Road",
+                                 "Crash/Road"
+                              )
+                           }
+                           icon={
+                              <Icon color="white" name="edit-road" size={50} />
+                           }
+                        />
+                     </HStack>
+                  )}
+               </Section>
+               <VStack>
+                  {vehiclesListArr}
+                  {this.state.edit && (
+                     <IconButton
+                        leftMargin={4}
+                        rightMargin={4}
+                        text="Add Vehicle"
+                        onPress={() => this._addVehicleSection()}
+                        icon={
+                           <Icon color="white" name="local-taxi" size={50} />
+                        }
+                     />
+                  )}
+               </VStack>
+               {this.state.edit ? (
```

Comment:
Do we want to move this check to the non motorist section? It is a redundant check that is already done in the nonmotist section.

Reply:
It is slightly redundant but also a little complex. Currently, the nonmotorist section represents a single nonmotorist, so adding this button may screw up the rendering. I would prefer to keep the add nonmotorist button separate.